graviti/manager/sheets.py

```diff
@@ -55,13 +57,18 @@ def _init_data(self) -> None:
             sheet = self._get_sheet(sheet_name)
             schema = PortexType.from_yaml(sheet["schema"])
 
+            patype = schema.to_pyarrow()
+
             factory = LazyFactory(
                 sheet["record_count"],
                 128,
                 partial(self._list_data, sheet_name=sheet_name),
-                schema.to_pyarrow(),
+                pa.struct([pa.field(RECORD_KEY, pa.string()), *patype]),
```

Comment:
do we need to do this change in search function?

Reply:
do we need `__record_key` in search result?

Reply:
There is no need for it